driver/src/db_interface.rs

```diff
@@ -124,12 +124,21 @@ impl DbInterface for MongoDB {
         ];
 
         info!("Querying standing_orders: {:?}", pipeline);
-        self.client
+        let non_zero_standing_orders = self.client
             .db(models::DB_NAME)
             .collection("standing_orders")
             .aggregate(pipeline, None)?
             .map(|d| d.map(models::StandingOrder::from).map_err(DriverError::from))
-            .collect::<Result<Vec<_>, _>>()
+            .collect::<Result<Vec<_>, _>>()?;
+
+        let empty_standing_order = models::StandingOrder::new(
```

Comment:
How about implementing `StandingOrder::empty()` ?

Reply:
Also, accountId shouldn't be always 0 but increase with each index, shouldn't it?